server/graphql-schema.js

```diff
@@ -689,6 +695,11 @@ const graphqlSchema = gql`
         results require this field to be filled in.
         """
         passed: Boolean
+        """
+        Indicates if this assertion result needs to be ignored because it is included with
+        a 0-level priority
+        """
+        exclude: Boolean
```

Comment:
If some assertions are not relevant, I don't think we should require users of the API to be responsible for detecting that and requiring them to pass in a special assertionResult with an "exclude" flag set to true. It feels like an example where complexity is escaping the abstraction. API consumers should only have to provide assertionResults which are relevant to the testResult.

Reply:
Addressed in https://github.com/w3c/aria-at-app/pull/863/commits/fe222f796e90e89d55bcf90c609bff3f566a0e22

Reply:
Looks good thanks for addressing that!!